components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -54,145 +57,202 @@ const TASKS = {
+            setStartPrebuildResult(result);
+        } catch (error) {
+            setPrebuildWasTriggered(false);
+            setEditorMessage(<EditorMessage type="warning" heading="Could not run prebuild." message={String(error).replace(/Error: Request \w+ failed with message: /, '')} />);
+        }
     }
-    try {
-      setPrebuildWasTriggered(true);
-      if (!isEditorDisabled) {
-        await getGitpodService().server.setProjectConfiguration(project.id, gitpodYml);
-      }
-      const result = await getGitpodService().server.triggerPrebuild(project.id, null);
-      setStartPrebuildResult(result);
-    } catch (error) {
-      setPrebuildWasTriggered(false);
-      setEditorMessage(<EditorMessage type="warning" heading="Could not run prebuild." message={String(error).replace(/Error: Request \w+ failed with message: /, '')}/>);
+
+    const onInstanceUpdate = (instance: WorkspaceInstance) => {
+        setPrebuildInstance(instance);
     }
-  }
 
-  const onInstanceUpdate = (instance: WorkspaceInstance) => {
-    setPrebuildInstance(instance);
-  }
+    useEffect(() => { document.title = 'Configure Project — Gitpod' }, []);
 
-  useEffect(() => { document.title = 'Configure Project — Gitpod' }, []);
+    return <>
+        <Header title="Configuration" subtitle="View and edit project configuration." />
+        <div className="lg:px-28 px-10 mt-8 flex space-x-4">
+            <div className="flex-1 h-96 rounded-xl overflow-hidden relative flex flex-col">
+                <div className="flex bg-gray-50 dark:bg-gray-800 border-b border-gray-200 dark:border-gray-600 px-6 pt-3">
+                    <TabMenuItem name=".gitpod.yml" selected={selectedEditor === '.gitpod.yml'} onClick={() => setSelectedEditor('.gitpod.yml')} />
+                    {!!dockerfile && <TabMenuItem name=".gitpod.Dockerfile" selected={selectedEditor === '.gitpod.Dockerfile'} onClick={() => setSelectedEditor('.gitpod.Dockerfile')} />}
+                </div>
+                {editorMessage}
+                <Suspense fallback={<div />}>
+                    {selectedEditor === '.gitpod.yml' &&
+                        <MonacoEditor classes="w-full flex-grow" disabled={isEditorDisabled} language="yaml" value={gitpodYml} onChange={setGitpodYml} />}
+                    {selectedEditor === '.gitpod.Dockerfile' &&
+                        <MonacoEditor classes="w-full flex-grow" disabled={isEditorDisabled} language="dockerfile" value={dockerfile} onChange={setDockerfile} />}
+                </Suspense>
+                {isDetecting && <div className="absolute h-full w-full bg-gray-100 dark:bg-gray-700 flex items-center justify-center space-x-2">
+                    {showAuthBanner ? (
+                        <div className="mt-8 text-gray-500 flex-col">
+                            <div className="p-6 text-center">
+                                <img src={NoAccess} title="No Access" className="m-auto mb-4" />
```

Comment:
**thought**: Thinking out loud, maybe we could use a more appealing or fun illustration here. What do you think of the following early draft? Yeah or [nah](https://en.wiktionary.org/wiki/nah#English)? Cc @jankeromnes 

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2021-09-29 at 2 09 43 PM (2)" src="https://user-images.githubusercontent.com/120486/135258628-6ab97969-1ba3-4a44-81db-182a25660c7e.png"> | <img width="1440" alt="Screenshot 2021-09-29 at 2 20 27 PM (2)" src="https://user-images.githubusercontent.com/120486/135258823-45057c3f-b574-4eaf-aec8-eec0d9ec4812.png"> |

Reply:
let's move such discussions to separate issues.

Reply:
Agree, added follow up issue in https://github.com/gitpod-io/gitpod/issues/5959. 🏓